src/igvfd/audit/measurement_set.py

```diff
@@ -471,3 +371,85 @@ def audit_missing_construct_library_set(value, system):
                 f'has no `construct_library_sets` of type {expected_library} linked in its `samples`.'
             )
             yield AuditFailure('missing construct library set', f'{detail} {description}', level='NOT_COMPLIANT')
+
+
+@audit_checker('MeasurementSet', frame='embedded')
+def audit_missing_auxiliary_set(value, system):
+    '''
+    [
+        {
+            "audit_description": "MPRA measurement sets are expected to link to a quantification DNA barcode sequencing auxiliary set.",
+            "audit_category": "missing auxiliary set",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "scQer MPRA measurement sets are expected to link to a circularized RNA barcode detection auxiliary set.",
+            "audit_category": "missing auxiliary set",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "CRISPR-based assay measurement sets, with the exception of mutagenesis assays, are expected to link to a gRNA sequencing auxiliary set.",
+            "audit_category": "missing auxiliary set",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "CRISPR-based measurement sets that utilize flow cytometry are expected to link to a cell sorting auxiliary set.",
+            "audit_category": "missing auxiliary set",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "Variant FlowFISH measurement sets are expected to link to a variant sequencing auxiliary set.",
+            "audit_category": "missing auxiliary set",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "10X Multiome MULTI-seq assay measurement sets are expected to link to a oligo-conjugated lipids auxiliary set.",
```

Comment:
Minor typo, "an oligo" instead of "a oligo"

Reply:
fixed.